enforest/operator.rkt

```diff
@@ -215,39 +215,48 @@
   #f)
 
 (define (apply-prefix-direct-operator op form stx track-origin checker)
-  (call-as-transformer
-   stx
-   track-origin
-   (lambda (in out)
-     (define proc (operator-proc op))
-     (out (checker (proc (in form) stx) proc)))))
+  (define proc (operator-proc op))
+  (checker
+   (call-as-transformer
+    stx
+    (list form)
+    track-origin
+    (lambda (form)
+      (proc form stx)))
+   proc))
 
 (define (apply-infix-direct-operator op form1 form2 stx track-origin checker)
-  (call-as-transformer
-   stx
-   track-origin
-   (lambda (in out)
-     (define proc (operator-proc op))
-     (checker (out (proc (in form1) (in form2) stx)) proc))))
+  (define proc (operator-proc op))
+  (checker (call-as-transformer
```

Comment:
Nit: the similar function call above is formatted differently.

Reply:
Thanks - made consistent.